pythonwhat/State.py

```diff
@@ -1,6 +1,6 @@
 import ast
 import inspect
-from pythonwhat.parsing import FunctionParser, ObjectAccessParser, ObjectAssignmentParser, IfParser, IfExpParser, WhileParser, ForParser, OperatorParser, ImportParser, FunctionDefParser, LambdaFunctionParser, ListCompParser, DictCompParser, GeneratorExpParser, WithParser, TryExceptParser
+from pythonwhat.parsing import FunctionParser, ObjectAccessParser, parser_dict
```

Comment:
I guess `FunctionParser` and `ObjectAccessParser` need special care because they don't give only one 'result' after parsing? Is there a way to do this more elegantly? I'd at least add some comments as to why you need a special treatment for these, otherwise it's pretty obscure.